.env_example

```
@@ -0,0 +1,8 @@
+SECRET_KEY='wok!18%)hf&_nxtnsm&z3eqvcub73iylmtbimk6v6--7-25fs-'
+DJANGO_SETTINGS_MODULE=raytro.settings.local
+PRODUCTION_HOST='raytro.herokuapp.com'
```

This should probably be changed to something more generic.
```suggestion
PRODUCTION_HOST='my.host.com'
```